volatility/framework/plugins/windows/crashinfo.py

```diff
@@ -22,21 +22,12 @@ def get_requirements(cls):
                                                      architectures = ["Intel32", "Intel64"]), 
             ]
         
-    def _generator(self, segments):
-        for seg in segments:
-            yield(0,(seg[0],seg[1],seg[2]))
+    def _generator(self, layer):
+        for seg in layer.mapping(0x0, layer.maximum_address, ignore_errors = True):
```

Comment:
Ideally, for readability this would be `for offset, length, mapped_offset, _, _ in ...`, referring to seg indices doesn't let people understand what's going on.